### ISSUE TITLE
Admin edit/change page for a single Resource 500s

### ISSUE THREAD

**Issue by lizadaly** [harvard-lil/h2o#1607]
Not sure if this is a Django 3.2 regression or if it's been there awhile; it's partially masked by #1540 because the Resource list page does not load on prod.

Going to a specific resource id in the admin returns:

```
NoReverseMatch at /django-admin/main/resource/232835/change/
Reverse for '_change' not found. '_change' is not a valid view function or pattern name.
```

Possibly because this is a proxy model. Other admin pages for normal models seem fine.

**Comment by lizadaly** [comment]
Now I'm wondering if we need this admin page at all, since `Resource` is just a superset of `LegalDocument`, `Link,`, `Section`, and `TextBlock`, which already have their own admin pages. Should we just delete this section of the admin?

**Comment by rebeccacremona** [comment]
When this worked, the `Resource` admin pages let a person inspect the `ContentNode`s that pointed to a `Case`, `Link` or `TextBlock`; there was a similar setup just for `Section` type `ContentNode`s and `Casebook` type `ContentNode`s. It was nice. Given the current broken model situation, it is less useful or not useful..... Maybe part of fixing/finishing the Giant Refactor is dropping both the `Section` and `Resource` admins, as well as getting `ContentNode` admin to work again.....

**Comment by lizadaly** [comment]
Was closed with #1935.
